ui-speedspacechart/src/components/helpers/drawElements/electricalProfile.ts

```diff
@@ -74,7 +74,6 @@ export const drawElectricalProfile = (
         for (let i = 0; i < 9; i++) {
           ctx.fillRect(x, topRect + 15 + i * 3, profileWidth, 1);
         }
-        ctx.stroke();
```

Comment:
Since we're getting rid of the various `stroke()` calls here in this file: I think we can get rid of `beginPath()` calls as well.

Reply:
Good point ! Done.